app/containers/Incentives/IncentiveSegmentContainer.tsx

```diff
@@ -15,37 +13,33 @@ const IncentiveSegmentContainer: React.FunctionComponent<Props> = ({
   const {
     productName,
     incentiveProduct,
+    incentiveProductMax,
     benchmark,
     eligibilityThreshold,
-    incrementalCarbonTax,
+    incentiveRatio,
+    incentiveMultiplier,
+    paymentAllocationFactor,
     emissionIntensity
   } = ciipIncentiveByProduct;
 
-  const formula = `
-  1 - \\left({ ${emissionIntensity} - ${benchmark}
-    \\over
-    ${eligibilityThreshold} - ${benchmark}
-  }\\right) \\times ${incrementalCarbonTax}`;
-
   return (
     <tr>
       <td>{productName}</td>
-      <td>
-        {process.env.NO_MATHJAX ||
-        getConfig()?.publicRuntimeConfig.NO_MATHJAX ? null : (
-          <MathJax.Context input="tex">
-            <MathJax.Node>{formula}</MathJax.Node>
-          </MathJax.Context>
-        )}
-      </td>
-      <td>CAD {incentiveProduct} </td>
+      <td>{Number.parseFloat(Number(emissionIntensity).toFixed(4))}</td>
```

Comment:
Are all of those `Number.parseFloat` needed?

Reply:
I added those for any of the values that we are rounding to 4 decimal places. The reason is if some of those values evaluate evenly to less decimal places, without the Number.parseFloat hanging zeroes are added to the end. example:
**0.1** becomes **0.1000**

Reply:
makes sense